torch/csrc/jit/script/module.h

```diff
@@ -75,7 +75,7 @@ struct TORCH_API Method {
     return function_->graph();
   }
 
-  const std::string& name() const {
+  std::string name() const {
```

Comment:
Why did this change? Is the result different? I assume this would be the unqualified name (since methods really don't have qualified names).

Reply:
Yeah, I think this was left over from something. Will change